examples/ColorTemperature/ColorTemperature.ino

```diff
@@ -70,16 +72,23 @@ void loop()
   if( (secs % DISPLAYTIME) < BLACKTIME) {
     memset8( leds, 0, NUM_LEDS * sizeof(CRGB));
   }
-  
+
   FastLED.show();
   FastLED.delay(8);
 }
 
 void setup() {
   delay( 3000 ); // power-up safety delay
+
   // It's important to set the color correction for your LED strip here,
   // so that colors can be more accurately rendered through the 'temperature' profiles
-  FastLED.addLeds<CHIPSET, LED_PIN, COLOR_ORDER>(leds, NUM_LEDS).setCorrection( TypicalSMD5050 );
+  // Uncomment this line for a 3-Wire chipset
+  FastLED.addLeds<LED_TYPE, DATA_PIN, COLOR_ORDER>(leds, NUM_LEDS).setCorrection( TypicalSMD5050 );
+
+  // Uncomment this line for a SPI chipset
```

Comment:
It can go either way.  If someone "says" SPI by sounding out the letters, it'd be an, eg "an S P I interface", however if someone pronounces SPI as a single word then a is ok "a spy interface" - since I have heard people use SPI in both ways in spoken conversation I'm unlikely to nitpick one way or the other.